test/functional/feature_maxuploadtarget.py

```
@@ -146,23 +168,32 @@ def run_test(self):
+        self.log.info("Restarting node 0 with download permission, bloom filter support and 1MB maxuploadtarget")
+        self.restart_node(0, ["-whitelist=download@127.0.0.1", "-peerbloomfilters", "-maxuploadtarget=1", "-blockmaxsize=999000", "-mocktime="+str(current_mocktime)])
+        # Total limit isn't reached after restart, but 1 MB is too small to serve historical blocks
+        self.assert_uploadtarget_state(target_reached=False, serve_historical_blocks=False)
 
         # Reconnect to self.nodes[0]
         peer = self.nodes[0].add_p2p_connection(TestP2PConn(), supports_v2_p2p=False)
 
+        # Sending mempool message shouldn't disconnect peer, as total limit isn't reached yet
+        peer.send_and_ping(msg_mempool())
+
         #retrieve 20 blocks which should be enough to break the 1MB limit
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)